staff/tests/forms/test_coupon_generation_form.py

```diff
@@ -0,0 +1,30 @@
+import csv
+import os
+import pytest
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from staff.forms.coupon_generation import CouponGenerationForm
+
+class TestCouponGenerationForm:
+    def test_empty_form_is_invalid(self):
+        coupon_generation_form = CouponGenerationForm(data={})
+
+        outcome = coupon_generation_form.is_valid()
+
+        assert outcome is False
+
+    def test_file_uploaded_field_accepts_csv(self):
```

Comment:
maybe the test name should reflect that we're testing for validation on incorrect file name?